src/main/kotlin/com/djvk/sarPdfParser/CsvWriter.kt

```diff
@@ -24,6 +25,9 @@ class CsvWriter(outFile: String, val docType: CsvHeaders.DocType) {
 
     fun insertRow(row: Map<String,String>){
         val list = getRowValuesInOrder(row)
+        list.map {
```

Comment:
I don't think map acts in place, does this do anything?

Reply:
It does indeed do the thing; not sure if I should change this to assign back to list to be more idomatic